ui/src/forms/FormProperty.vue

```diff
@@ -39,7 +39,7 @@ export default class extends Vue {
   @Prop() renderMultiEditor!: () => TemplateResult;
 
   get isHidden (): boolean {
-    return !!this.property.shape.pointer.out(dash.hidden).value
+    return this.property.shape.hidden === true
```

Comment:
Is the `=== true` part necessary?

Reply:
Technically not but `hidden` returns `boolean | undefined` which is not exactly the same as strict truth

Reply:
Might as well change the return type of `isHidden`

Reply:
I would keep it that way then. Boolean is a better return type than `boolean | undefined`.